packages/css-library/dist/tokens/json/variables.json

```diff
@@ -4177,20 +4177,5 @@
     "path": [
       "uswds-system-color-mint-cool-50"
     ]
-  },
-  "uswds-system-color-red-cool-10v": {
```

Comment:
I'm curious why this color needed to be removed? Is cleanup from a previous tokens ticket we had been working on?

Reply:
Yes its cleanup, the build was failing because this variable didn't exist in USWDS. It needed to be "uswds-system-color-red-cool-vivid-10"